vendor/engines/saml_authentication/lib/saml_authentication/devise_configurator.rb

```diff
@@ -9,8 +9,8 @@ def configure!
       Devise.setup do |config|
         config.saml_session_index_key = :session_index
         config.saml_default_user_key = :username
-        config.saml_create_user = true
-        config.saml_update_user = true
+        config.saml_create_user = SettingsHelper.feature_on?(:saml_create_user)
+        config.saml_update_user = SettingsHelper.feature_on?(:saml_create_user)
```

Comment:
@jhanggi I know we explicitly don't want to update the email, but we seem to be depending on this to do updating of the attributes as well.

The specs need updating to take into account the feature being on and off, obviously, but I'd like to talk through with you what we want to be the expected behavior for each state when you get a chance.

Reply:
Do this for only create, move from feature flag into the saml settings.